props.json

```diff
@@ -1,4 +1,147 @@
 {
+  "1.7.0": {
+    "whatsNewUrl": "https://wolvic.com/blog/release_1.6.1",
```

Comment:
Is this link correct ?

Reply:
Updated to: https://wolvic.com/blog/release_1.7